jsonnet/main.jsonnet

```diff
@@ -181,6 +181,7 @@ local inCluster =
             rateInterval: '1m',  // adjust the rate interval value to be 4 x the node_exporter's scrape interval (15s).
           },
         },
+        ignoredNetworkDevices:: '^(veth.*|[a-f0-9]{15}|enP.*)$',
```

Comment:
Can you add a comment explaining the reasons for each exclusion (e.g. like we have in kube-prometheus)?

Reply:
thanks, explanation added :)

Reply:
cool, thanks!